app/src/layouts/tabular/index.ts

```diff
@@ -154,6 +154,13 @@ export default defineLayout<LayoutOptions, LayoutQuery>({
 			selection.value = items.value.map((item) => item[pk.field]);
 		}
 
+		function calculateDefaultWidth(fieldName: string, minWidth = 100, maxWidth = 400): number {
+			const avgCharWidth = 12;
+			const padding = 24;
+			const baseWidth = fieldName.length * avgCharWidth + padding;
```

Comment:
`fieldName.length` is not going to work well here with relationals, in the example I tested, this equates to `labels.labels_id.name`

Which gets you ~260px 👇 

![Screenshot 2025-06-19 at 11 37 29 AM](https://github.com/user-attachments/assets/05516089-3a06-4487-84ec-db8fb72f4af4)